Dockerfile

```diff
@@ -62,7 +62,15 @@ RUN cp $(python3 -m ara.setup.callback_plugins)/*.py /usr/share/ansible/plugins/
 
 # Add community modules and pre-install necessary binaries to support them from the distro
 RUN curl -L --output sops.rpm ${SOPS_RPM_SOURCE} && rpm -ivh sops.rpm
-RUN ansible-galaxy collection install community.sops:$COMMUNITY_SOPS_VERSION
+RUN ansible-galaxy collection install community.sops:$COMMUNITY_SOPS_VERSION && \
+    ansible-galaxy collection install community.general && \
+    ansible-galaxy collection install community.hashi_vault:5.0.0 && \
```

Comment:
Is there a reason to pin this version for hashi_vault and not the others?

Reply:
Ideally they should all be pinned, and then exposed as docker ARGs so that we can iterate on them as necessary, just as the COMMUNITY_SOPS_VERSION is. My goal is to unblock first; iterating on some of the details can wait.